src/Codec/Xlsx/Parser.hs

```diff
@@ -136,69 +156,65 @@ extractCellValue _ "b" "0" = [CellBool False]
 extractCellValue _ _ _ = []
 
 -- | Get xml cursor from the specified file inside the zip archive.
-xmlCursor :: Zip.Archive -> FilePath -> Maybe Cursor
-xmlCursor ar fname = parse <$> Zip.findEntryByPath fname ar
+xmlCursor :: Zip.Archive -> FilePath -> Parser (Maybe Cursor)
+xmlCursor ar fname = maybe (Right Nothing) parse $ Zip.findEntryByPath fname ar
```

Comment:
what about use `note` in `xmlCursorRequired` and `hush` to transform it into `Maybe` in this function? We have `Either` from `parseLBS` and it doesn't look cool to turn it into `Maybe` and then back into `Either`